readwrite/main.go

```diff
@@ -18,7 +17,7 @@ func writeFile() {
 	}
 }
 func readFile() {
-	data, err := ioutil.ReadFile("Student.txt")
+	data, err := os.ReadFile("Student.txt")
```

Comment:
Did you do this for below reason?

```md
Deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.
```

Source: https://pkg.go.dev/io/ioutil

Reply:
Yes